Sources/QuickCheck/Gen.swift

```diff
@@ -45,6 +45,12 @@ public func resize<A>(_ size: Int) -> (Gen<A>) -> Gen<A> {
   }
 }
 
+public func scale<A>(_ f: @escaping (Int) -> Int) -> (Gen<A>) -> Gen<A> {
```

Comment:
I'd used this in the Haskell version. Not sure why the PureScript version doesn't define it!